### ISSUE TITLE
Enable extension to work on Kiwi browser

### ISSUE THREAD

**Issue by jepsefan** [sniklaus/youtube-watchmarker#135]
I am using Kiwi browser om android and it supports chrome extension. I installed it and it didn't work because https://m.youtube.com/ is not available but https://youtube.com/ is, so if i force desktop mode it works.
Could you add mobile web page in extension settings.

**Comment by sniklaus** [comment]
Thanks for bringing this up! This seems to be a duplicate of #102 which is also asking for mobile support?

**Comment by sniklaus** [comment]
I just released version `4.7.0` (already live on Firefox, still pending on Chrome) which also supports `m.youtube.com` though I didn't thoroughly test it so your mileage may vary. I am hence closing this issue for now but please feel free to reopen it anytime should something be broken.
